tests/integration/security/rbac/rbac_v2_test.go

```diff
@@ -38,22 +40,28 @@ const (
 )
 
 var (
-	inst                 istio.Instance
-	successIfAuthEnabled bool
+	inst               istio.Instance
+	allowIfAuthEnabled bool
 )
 
 func TestMain(m *testing.M) {
-	framework.Main("rbac_v2_test", m, istio.SetupOnKube(&inst, setupConfig))
+	framework.
+		NewSuite("rbac_v2", m).
+		// TODO(pitlv2109: Turn on the presubmit label once the test is stable.
+		//Label(label.Presubmit).
+		RequireEnvironment(environment.Kube).
+		Setup(istio.SetupOnKube(&inst, setupConfig)).
+		Run()
 }
 
 func setupConfig(cfg *istio.Config) {
 	if cfg == nil {
 		return
 	}
 	if cfg.IsMtlsEnabled() {
-		successIfAuthEnabled = true
+		allowIfAuthEnabled = true
```

Comment:
`allowIfAuthEnabled = cfg.IsMtlsEnabled()`, actually it's better to rename `allowIfAuthEnabled` to `isMtlsEnabled`.

Reply:
Thanks, will change to `allowIfAuthEnabled = cfg.IsMtlsEnabled()`. As for the second point, I don't think reading `IsAllowed: isMtlsEnabled` makes much sense than `IsAllowed: allowIfAuthEnabled`

Reply:
`isMtlsEnabled` clearly indicates what it is, when looking at the actual test cases that use it, it's clear what you're asserting at.
Compare the following two forms:
`IsAllowed: allowIfAuthEnabled` and `IsAllowed: isMtlsEnabled`, The second one is much more clear to infer that it's allowed if mtls is enabled.

Reply:
I don't see much difference tbh, but I will change it to `isMtlsEnabled` :)